### DISCUSSION TITLE
Hardware: Passively-cooled server, 16 Cameras with restreaming?

### DISCUSSION THREAD

**Discussion by oguruma1218** [blakeblackshear/frigate#6412] [Ask A Question]
I have 16 1080P Cameras. I'd like to build a frigate box that can uses a passively-cooled CPU, and also support restreaming. I have a Coral I'll use for the object detection. 

Is it possible to run Frigate with 16 cameras with any of the available low-power CPUs?

**Comment by LaurenceGough** [comment]
Anything is possible if you have a large enough passive cooler ;). 

I am only at 10-15% with 2 cameras, one with a full HD detect stream with just a low powered Intel N5100 CPU (+ USB CORAL), it does have a very low powered laptop fan but it could be easily passively cooled which I will be doing later. The H.265 camera does use a bit more processing power too.  Frigate says only around 1.5% GPU usage on my system.

I believe that CPU usage would only be impacted by motion detection and object detection on the detect streams, which is usually a substream so this might even be lower than 1080p.

It might be worth looking at the latest Intel processors with a low manufacturing node, so they run nice and efficient. The hardware video decoding (quick sync) plays very well with Frigate.

Here's a few to check out.

Intel Pentium Gold https://www.intel.com/content/www/us/en/products/details/processors/pentium/gold/products.html
https://www.intel.com/content/www/us/en/products/sku/230903/intel-celeron-processor-7305l-8m-cache-1-10-ghz/specifications.html
Intel Processor N200
Intel Processor U300 (might get a bit toasty perhaps)
There are also the 12th gen i3s you can take a look at.

People are running far more powerful systems off with massive case heatsinks on desktop towers so these should all be fine.

**Comment by jaco** [comment]
I have 4x 4mp cameras on N6005 without fan but for now (it’s temporary setup) detection runs only on 720p and load is 0.2. This comes from ffmpeg decode. 

The plan is to run 6x 8mp cameras with detection on minimum 1080p.

**Comment by FrigateEnthusiast** [comment]
Don't forget about your Coral cooling. If hammering it with non-stop inferencing from 16 cams it could reach it's thermal throttling point. Read this:

Manage the PCIe module temperature
"https://coral.ai/docs/pcie-parameters/"

I put small solid copper heatsinks on each chip like they suggest.
